src/System.Net.Sockets/tests/FunctionalTests/SendPacketsAsync.cs

```diff
@@ -279,6 +279,26 @@ public void BufferZeroCountThenNormal_ZeroCountIgnored(SocketImplementationType
 
         #endregion Buffers
 
+        #region TransmitFileOptions
+        [Theory]
+        [InlineData(SocketImplementationType.APM)]
+        [InlineData(SocketImplementationType.Async)]
+        [PlatformSpecific(TestPlatforms.Windows)]
+        public void SocketDisconnected_TransmitFileOptionDisconnect(SocketImplementationType type)
+        {
+            SendPackets(type, new SendPacketsElement(new byte[10], 4, 4), TransmitFileOptions.Disconnect, 4);
+        }
+
+        [Theory]
+        [InlineData(SocketImplementationType.APM)]
+        [InlineData(SocketImplementationType.Async)]
+        [PlatformSpecific(TestPlatforms.Windows)]
+        public void SocketDisconnectedAndReusable_TransmitFileOptionReuseSocket(SocketImplementationType type)
+        {
+            SendPackets(type, new SendPacketsElement(new byte[10], 4, 4), TransmitFileOptions.Disconnect | TransmitFileOptions.ReuseSocket, 4);
+        }
```

Comment:
The test for that already exists.. Source code [here](https://github.com/dotnet/corefx/blob/master/src/System.Net.Sockets/tests/FunctionalTests/SendPacketsAsync.cs#L61)